src/SampleWebApp/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Country.g.cs

```diff
@@ -88,4 +89,17 @@ public partial class Country : AuditableEntityBase
     /// The top level internet domains regitered to the country (comma-delimited) (optional).
     /// </summary>
     public Text? TopLevelDomains { get; set; } = null!;
+    
+    /// <summary>
+    /// Country accepts as legal tender OneOrMany Currencies
+    /// </summary>
+    public virtual List<Currency> Currencies { get; set; } = new List<Currency>();
+    
+    [NotMapped]
```

Comment:
we are not using attributes, the setup of the relation should be enough, be cause there we state its a navigation property